sdk/src/main/java/com/oursky/authgear/latte/Latte.kt

```diff
@@ -365,4 +390,14 @@ class Latte(
             page = latteOptions.page
         )
     }
+
+    private suspend fun makeAuthgearReauthenticateOptions(latteOptions: ReauthenticateOptions): com.oursky.authgear.ReauthentcateOptions {
```

Comment:
com.oursky.authgear.ReauthentcateOptions has typo.

Reply:
@louischan-oursky Yes I saw that too, but If we change it now I think it will become a breaking change. Not sure if we want to fix it at the moment.

Reply:
But people should be able to choose the sdk version to install, so maybe I just fix it now.